optuna/multi_objective/study.py

```diff
@@ -48,6 +48,24 @@ def create_study(
 ) -> "multi_objective.study.MultiObjectiveStudy":
     """Create a new :class:`~optuna.multi_objective.study.MultiObjectiveStudy`.
 
+    Example:
+
+            .. testcode::
+
+                import optuna
+
+                def objective(trial):
+                    # Binh and Korn function.
+                    x = trial.suggest_float("x", 0, 5)
+                    y = trial.suggest_float("y", 0, 3)
+
+                    v0 = 4 * x ** 2 + 4 * y ** 2
+                    v1 = (x - 5) ** 2 + (y - 5) ** 2
+                    return v0, v1
+
+                study = optuna.multi_objective.create_study(["minimize", "minimize"])
+                study.optimize(objective, n_trials=3)
```

Comment:
It seems the indent is too deep. Could you reduce them?
```suggestion
    Example:

        .. testcode::

            import optuna

            def objective(trial):
                # Binh and Korn function.
                x = trial.suggest_float("x", 0, 5)
                y = trial.suggest_float("y", 0, 3)

                v0 = 4 * x ** 2 + 4 * y ** 2
                v1 = (x - 5) ** 2 + (y - 5) ** 2
                return v0, v1

            study = optuna.multi_objective.create_study(["minimize", "minimize"])
            study.optimize(objective, n_trials=3)
```

Reply:
Thank you for suggestion!